### PR TITLE
Pages

### LINKED ISSUE #11: Home page
# User-Story

As a user

I want to have a home page

so that i can see everything at once

# Notes, Links, Material
image

# ToDos

- [x] Add header #4 
- [x] Add categories #5 
- [x] Add footer #3

### LINKED ISSUE #12: Category Page
# User-Story

As a user

I want to have a category page

so that i can see projects from a specific category

# Notes, Links, Material
image

# ToDos

- [x] Add header #4 
- [x] Add footer #3 
- [x] Add example projects #6

### LINKED ISSUE #13: Browse Page
# User-Story

As a user

I want to a browse projects page

so that i can search for specific projects and tags

# Notes, Links, Material
image

# ToDos

- [x] Add header #4 
- [x] Add footer #3 
- [x] Add searchbar #7

### LINKED ISSUE #14: Add project page
# User-Story

As a user

I want to have a Add a project page

so that i can add my own projects

# Notes, Links, Material
image

# ToDos

- [x] Add header #4 
- [x] Add footer #3 
- [x] Add buttons #8 
- [x] Add input  fields #9

### LINKED ISSUE #15: Favourites page
# User-Story

As a user

I want to have a favourites page

so that i can see all my favourite projects at once

# Notes, Links, Material
image

# ToDos

- [x] Add header #4 
- [x] Add footer #3
- [x] Add example favorites #6

### LINKED ISSUE #16: Details page
# User-Story

As a user

I want to have a details page

so that i can see photo, title and description of the project

# Notes, Links, Material
image

# ToDos

- [x] Add header #4
- [x] Add footer #3 
- [x] Add example project image and text

### PR DESCRIPTION
**Pull request by kschmitz123**
I've added all my pages in Storybook. They do not have any functionality yet, just styling.

close #11 
close #12 
close #13 
close #14 
close #15 
close #16

### CODE REVIEW

File: client/src/GlobalStyle.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ const GlobalStyle = createGlobalStyle`
 }
 
 body,
-button {
+button, textarea {
```
Comment (phlgr) [inline client/src/GlobalStyle.js]: Do you really want to change all buttons and textareas? Why don't you build a component where you remove said properties and branch out new Buttons from there?
html (baseline button) => styled Button
html => baseline Button => styled Button

File: client/src/components/Categories.js [hunk 2]
```diff
@@ -33,9 +33,13 @@ const Container = styled.div`
   flex-wrap: wrap;
 `;
 
+const Div = styled.div`
```
Comment (phlgr) [inline client/src/components/Categories.js]: This is not a good name, instead call it `CategoryContainer` or similiar

File: client/src/components/Input.js [hunk 3]
```diff
@@ -16,31 +17,37 @@ const SmallInput = styled.input`
...
 
-const LargeInput = styled.input`
-  height: 40rem;
+const LargeInput = styled.textarea`
+  height: 20rem;
   border-radius: 15px;
   width: 80%;
   border: solid 1px var(--secondary-font-color);
-  text-align: center;
 `;
```
Comment (phlgr) [inline client/src/components/Input.js:25]: Are these specific to the Form or can they be used in other components as well? Maybe create a separate component for these.

File: client/src/components/ScrollMenu.js [hunk 4]
```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import styled from "styled-components/macro";
+import Test from "../assets/test.jpg";
+
+const Div = styled.div`
```
Comment (phlgr) [inline client/src/components/ScrollMenu.js]: Naming, also try finding your own naming conventions, sometimes you name the outer Container `Div` sometimes `Container`

File: client/src/pages/CategoryPage.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import styled from "styled-components/macro";
+import { Header } from "../components/Header";
+import { ImagePreview } from "../components/ImagePreview";
+import { Navbar } from "../components/Navbar";
+
+const Container = styled.div`
+  padding: 60px 10px;
+`;
```
Comment (phlgr) [inline client/src/pages/CategoryPage.js]: Maybe try adding a general `Container` component, so you don't have to create a new Container Component for every Component

File: client/src/pages/FavouritesPage.js [hunk 6]
```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import styled from "styled-components/macro";
+import { Header } from "../components/Header";
+import { ImagePreview } from "../components/ImagePreview";
+import { Navbar } from "../components/Navbar";
+
+const Container = styled.div`
+  padding: 60px 10px;
+`;
```
Comment (phlgr) [inline client/src/pages/FavouritesPage.js]: General Container Component

**Review by phlgr — CHANGES_REQUESTED** [review]
Please look into my changes. Feel free to ask for explanations about the requested changes

File: client/src/GlobalStyle.js (hunk 1 above)
Comment (kschmitz123) [reply client/src/GlobalStyle.js:17]: I actually just set font and color which i do want for all buttons and textareas :)

File: client/src/components/Input.js (hunk 3 above)
Comment (kschmitz123) [reply client/src/components/Input.js:25]: they are specific to this form :)

File: client/src/pages/FavouritesPage.js [hunk 7]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
```
Comment (mmurakaru) [inline client/src/pages/FavouritesPage.js:1]: I think with the latest react version -import React from 'react'- is done by default / consider removing this

File: client/src/pages/HomePage.js [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+      <Header title={"Craftified"} />
+      <SearchButton>
+        <SearchIcon fontSize="large" />
+      </SearchButton>
+      <ScrollMenu />
+      <Categories />
+
+      <Navbar />
+    </>
+  );
```
Comment (mmurakaru) [inline client/src/pages/HomePage.js:33]: Consider puting your css below the component to create some kind of a hierarchy in first reads. imports > function component > styles. 

Would it possibly make sense to make the SearchButton a child and relative to your header with position: absolute?

**Pull request merged**